drivers/trim_util.py

```diff
@@ -111,15 +111,23 @@ def do_trim(session, args):
 
             # Clean trim LV in case the previous trim attemp failed
             if lvutil.exists(lv_path):
-                lvutil.remove(lv_path)
-
-            # Perform a lvcreate, blkdiscard and lvremove to trigger trim on the array
-            lvutil.create(lv_name, 0, vg_name, size_in_percentage="100%F")
-            cmd = ["/usr/sbin/blkdiscard", "-v", lv_path]
-            stdout = util.pread2(cmd)
-            util.SMlog("Stdout is %s" % stdout)
-            util.SMlog("Trim on SR: %s complete. " % sr_uuid)
-            result = str(True)
+               lvutil.remove(lv_path)
+
+            #Check if VG limits are enough for creating LVM.
+            stats = lvutil._getVGstats(vg_name)
+            if (stats['freespace'] < lvutil.LVM_SIZE_INCREMENT):
+                util.SMlog("No space to claim on a completely filled SR %s to perform lvutil.create" % sr_uuid)
```

Comment:
Simplify the comment. "No space to claim on SR to perform lvutil.create" is a little complicated. 
Also, for this and the next couple of lines
- Limit line length to 80 characters. 
- Change "completely filled SR" to "full SR"